src/main/resources/static/app.js

```diff
@@ -0,0 +1,151 @@
+let stompClient = null;
+
+function setConnected() {
+    $("#buttonConnect").addClass("d-none");
+    $("#buttonDisconnect").removeClass("d-none");
+    $("#buttonConnecting").addClass("d-none");
+
+    $("#alertBox").removeClass();
+    $("#alertBox").addClass("alert alert-success");
+    $("#alertBox").text("Connected :)");
+}
+
+function setDisconnected() {
+    $("#buttonConnect").removeClass("d-none");
+    $("#buttonDisconnect").addClass("d-none");
+    $("#buttonConnecting").addClass("d-none");
+
+    $("#alertBox").removeClass();
+    $("#alertBox").addClass("alert alert-warning");
+    $("#alertBox").text("Server not connected!");
+}
+
+function setConnectionPending() {
+    $("#buttonConnect").addClass("d-none");
+    $("#buttonDisconnect").addClass("d-none");
+    $("#buttonConnecting").removeClass("d-none");
+}
+
+function connect() {
+    setConnectionPending();
+
+    let socket = new SockJS('/gs-guide-websocket');
+    stompClient = Stomp.over(socket);
+    stompClient.connect(
+        {}, // headers
+        function (frame) { // connectCallback
+            setConnected();
+            console.log('Connected: ' + frame);
+
+            stompClient.subscribe('/gol/generation', function (generationData) {
```

Comment:
Do not abbreviate `gol`. To be most simple, we could just remove `/gol`, as the whole package and project is named `gameoflife`, anyway.

Also consider the partnering endoints below.